File: distros/J/JavonetPerlSdk/lib/Javonet/Binaries/Nodejs/core/handler/ArrayHandler.js
const AbstractHandler = require("./AbstractHandler")


class ArrayHandler extends AbstractHandler {
    constructor() {
        super()
    }

    process(command) {
        try {
            let processedArray = command.payload
            return processedArray
        } catch (error) {
            throw this.process_stack_trace(error, this.constructor.name)
        }
    }
}

module.exports = new ArrayHandler()

File: distros/J/JavonetPerlSdk/lib/Javonet/Binaries/Python/javonet/core/handler/ArrayHandler.py
from javonet.core.handler.AbstractCommandHandler import AbstractCommandHandler


class ArrayHandler(AbstractCommandHandler):
    def process(self, command):
        try:
            processedArray = command.payload
            return processedArray

        except Exception as e:
            exc_type, exc_value = type(e), e
            new_exc = exc_type(exc_value).with_traceback(e.__traceback__)
            raise new_exc from None
